.circleci/config.yml

```diff
@@ -196,5 +253,5 @@ workflows:
           filters: { branches: { ignore: develop } }
 
       - publish:
-          requires: [ unit-test, unit-test-14, check, trial-publish ]
+          requires: [ unit-test, unit-test-11, unit-test-15, unit-test-14, check, trial-publish ]
```

Comment:
I just re-rendered this by running `REPOSITORY_OWNER=palantir REPOSITORY_NAME=tritium /Volumes/git/circle-templates/excavator/circleci/render.sh oss`... might actually just flip the `unit-test-14` thing from being default on to unit-test-15 being default on

Reply:
yeah, I would prefer to drop JDK 14 when we can given it is end of life